src/prices/repository.js

```diff
@@ -52,22 +52,23 @@ async function findPrice(db, priceId) {
         throw new Error("tried to find a price", error);
     }
 
-    let price = {
+    let dayPrices = {
         id: result.rows[0].id,
-        data: result.rows[0].data
-        // monday: result.rows[0].data.monday,
-        // tuesday: result.rows[0].data.tuesday,
-        // wednesday: result.rows[0].data.wednesday,
-        // thursday: result.rows[0].data.thursday,
-        // friday: result.rows[0].data.friday,
-        // saturday: result.rows[0].data.saturday,
-        // sunday: result.rows[0].data.sunday
+        data: {
+            monday: result.rows[0].data.monday,
+            tuesday: result.rows[0].data.tuesday,
+            wednesday: result.rows[0].data.wednesday,
+            thursday: result.rows[0].data.thursday,
+            friday: result.rows[0].data.friday,
+            saturday: result.rows[0].data.saturday,
+            sunday: result.rows[0].data.sunday
+        }
```

Comment:
There are two approaches to reducing the redundant code (`result.rows[0].data`) here...

1. [lodash pick](https://www.npmjs.com/package/lodash.pick) can be used to do `data: pick(result.rows[0].data, [ "monday", "tuesday", "wednesday", "thursday", "friday", "saturday", "sunday" ])`

2. Pick using destructuring before building the object: `let { monday, tuesday, wednesday, thursday, friday, saturday, sunday } = result.rows[0].data;` and then adding them in the dayPrices object as `data: { monday, tuesday, wednesday, thursday, friday, saturday, sunday }`.

Reply:
Oops, just noticed that this is the prices repo that is removed again in next commit. But the feedback stands :smile:

Reply:
very nice, thanks! I changed it in the priceRepo in another branch 😄